### ISSUE TITLE
v0.001 test suite: Debug Info dialog not opening/parsing properly

### ISSUE THREAD

**Issue by pryrt** [pryrt/Win32-Mechanize-NotepadPlusPlus#4]
http://www.cpantesters.org/cpan/report/eef566ca-6bfd-1014-8029-d8f6252935ce

I'm not sure that the test suite bombs out with an appropriate message if Notepad++ isn't found running or in the path.  I need to investigate that.

**Comment by pryrt** [comment]
According to my experiments, it _does_ bomb out if Notepad++ is not found.

```
perl -Ilib -MWin32::Mechanize::NotepadPlusPlus=:main -MData::Dumper -le "print Dumper notepad()"

Use of uninitialized value in sprintf at lib/Win32/Mechanize/NotepadPlusPlus/Notepad.pm line 176.
could not find an instance of notepad++; please add it to your path at lib/Win32/Mechanize/NotepadPlusPlus/Notepad.pm line 245.
BEGIN failed--compilation aborted at lib/Win32/Mechanize/NotepadPlusPlus.pm line 49.
Compilation failed in require.
BEGIN failed--compilation aborted.
```

and 

```
prove -vl t\00-load.t
t\00-load.t ..
1..7
Use of uninitialized value in sprintf at C:\usr\local\share\GitHubSvn\Win32-Mechanize-NotepadPlusPlus\lib/Win32/Mechanize/NotepadPlusPlus/Notepad.pm line 176.

#   Failed test 'use Win32::Mechanize::NotepadPlusPlus;'
#   at t\00-load.t line 19.
#     Tried to use 'Win32::Mechanize::NotepadPlusPlus'.
not ok 1 - use Win32::Mechanize::NotepadPlusPlus;#     Error:  could not find an instance of notepad++; please add it to your path at C:\usr\local\share\GitHubSvn\Win32-Mechanize-NotepadPlusPlus\lib/Win32/Mechanize/NotepadPlusPlus/Notepad.pm line 245.
# BEGIN failed--compilation aborted at C:\usr\local\share\GitHubSvn\Win32-Mechanize-NotepadPlusPlus\lib/Win32/Mechanize/NotepadPlusPlus.pm line 49.
# Compilation failed in require at t\00-load.t line 19.
# BEGIN failed--compilation aborted at t\00-load.t line 19.
# Couldn't even load Win32::Mechanize::NotepadPlusPlus
```

If notepad++ is in the path, or if it's already running, then it works; though if it's not in the path but running, then the test sequence will die when it hits the 30/31 sequence (just confirmed), because those close the active Notepad++.

**Comment by pryrt** [comment]
Looking at the error message more closely: it appears there was a copy of Notepad++, but perhaps it wasn't English localization, so it never found the `Debug Info` dialog box.

Hmm, it might also be because the appropriate window didn't take focus, since `'right foreground window'` was failing

Two things I want to do to help debug this:
1. in 00-load, `diag` the executable path (possibly trimmed down to avoid accidentally revealing too much about a smoke-tester's computer)
2. in 01-bits, `diag` the list of child windows.

**Comment by pryrt** [comment]
Per the v0.001001 [matrix entry](http://www.cpantesters.org/cpan/report/0d80104c-6c89-1014-af95-4e2c2aeac5ab) by the same tester, it's made it past 02-bits.t in the new test suite.  There's still a bug, but I think it's unrelated, so will be starting a new issue or two (#10, #11).

Assuming v0.001001 Closed #4.
